index.html

```diff
@@ -9,20 +9,24 @@
 
     <section></section>
     </head>
+    <section><img src="photos/photoNicole.jpeg" alt="Nicole" width="300" height="300" style="border-radius: 50%"></section>
     <section>
-        <h1>ABOUT:</h1>
-        <h2>Hi, my name is Nicole Bradley. I love computer coding. It is my dream to one day be a game developer. I am playing a game called Star Trek Fleet Command. I have to play the games I want to learn how to make. </h2>
+        <h1 style="color:purple;">ABOUT:</h1>
+        <h2>Hi, my name is Nicole Bradley. I love computer coding. It is my dream to one day be a game developer. 
+            I am playing a game called Star Trek Fleet Command. 
+            I have to play the games I want to learn how to make. </h2>
     </section>
 
     <section>
-     <h1>EXPERIENCE:</h1>
-     <h2>I have 4 years experience in coding, AND I am woking on a project called "Greetings World". It is in progress though.</h2>
+        <h1 style="color:purple;">EXPERIENCE:</h1>
+        <h2>I have 4 years experience in coding, and I am working on a project called "Greetings World". 
+        It is a social media site for greeting cards, set up like an email system. The project is in progress.</h2>
     </section>
 
     <section>
-        <h1>CONNECT:</h1>
-        <p><a>https://github.com/nicoleCodeGirl/greetingsWorld</a></p>
-        <p><a>https://www.twitch.tv/nicolecodegirl</a></p>
+        <h3 style="color:purple;">CONNECT:</h3>
+        <p><a href="https://github.com/nicoleCodeGirl/greetingsWorld">https://github.com/nicoleCodeGirl/greetingsWorld</a></p>
+        <p><a href="https://www.twitch.tv/nicolecodegirl">https://www.twitch.tv/nicolecodegirl</a></p>
```

Comment:
```suggestion
        <p><a href="https://github.com/nicoleCodeGirl/greetingsWorld">Github</a></p>
        <p><a href="https://www.twitch.tv/nicolecodegirl">Twitch</a></p>
```
The visible text for a link usually doesn't need to be the full URL